modcc/astmanip.hpp

```diff
@@ -10,19 +10,30 @@
 
 // Create new local variable symbol and local declaration expression in current scope.
```

Comment:
I can muck about with the comments (I'll submit a tiny patch later), but the current layout is more, not less, robust to become stale with refactoring. The return type is part of the function signature, and has no independent life from the function itself — it exists, essentially, only to be returned. If the interface to the function changed, the comment would remain unchanged.